src/views/workspaces/login/signup.js

```diff
@@ -182,7 +184,6 @@ export const WorkspaceSignupView = withSocialSignupAllowed(({
                             </Paragraph>
                         </div>
                     }
-                    logo={ <AppstoreOutlined /> }
```

Comment:
Again confused why we're deleting this logo; are we not happy with how it looks or is it visual clutter or something else? I'd prefer we replace it if we just don't like the appearance.

Reply:
The hope was to make that logo to be something else and not be appstore specific.